gesturelibrary/include/gesturelib.h

```diff
@@ -78,7 +90,3 @@ int enable_recognizer(int recognizer);
 /// @param recognizer which recognizer to disable
 /// @return 0 if an invalid index is provided, 1 otherwise
 int disable_recognizer(int recognizer);
-
-/// @brief
-/// @return the number of registered recognizers
-int get_num_recognizers();
```

Comment:
Why was this removed?

Reply:
It shouldn't be removed. I'll look at that again when I get back home

Reply:
Oh I remember now. It's because I exposed the `num_recognizers` variable as extern within the header file. Didn't need to have a getter for it.